### PR TITLE
Add text sources to FAQ

### PR DESCRIPTION
**Pull request by DeathCamel57**
In re: #287 and #306

### CODE REVIEW

File: web/src/lib/components/pages/faq.tsx [hunk 1]
```diff
@@ -30,6 +30,10 @@ export default class FAQ extends Component<Props, State> {
 
       <h3>Why is 10,000 hours the goal for capturing audio?</h3>
       <p>This is approximately the number of hours required to train a production STT system.</p>
+
+      <h3>Where can I find the sample text?</h3>
+      <p>Common Voice is an open source project that is on <a href="https://github.com/mozilla/voice-web/">Github</a></p>
+      <p>Our sets of samples are available <a href="https://github.com/mozilla/voice-web/blob/master/server/data/temporary-sentences.txt">here</a>, <a href="https://github.com/mozilla/voice-web/blob/master/server/data/temporary-sentences-2.txt">here</a>, <a href="https://github.com/mozilla/voice-web/blob/master/server/data/funny.txt">here</a>, and <a href="https://github.com/mozilla/voice-web/blob/master/server/data/new-sentences.txt">here</a>.</p>
```
Comment (mikehenrty) [inline web/src/lib/components/pages/faq.tsx]: Right now, we just use `https://github.com/mozilla/voice-web/blob/master/server/data/new-sentences.txt`.

**Comment by mikehenrty** [comment]
To fix bug #330, you will also have to add where these sentences come from, which is War of the Worlds and The Alchemist.

**Comment by DeathCamel57** [comment]
What about this edit @mikehenrty

File: web/src/lib/components/pages/faq.tsx (hunk 1 above)
Comment (DeathCamel57) [inline web/src/lib/components/pages/faq.tsx]: Fixed!

**Comment by mikehenrty** [comment]
Unfortunately, I just changed our source material and where it lives in the tree. I'll accept your change and then patch it.

**Pull request merged**